internal/rbd/rbd_attach.go

```diff
@@ -313,6 +327,9 @@ func appendRbdNbdCliOptions(cmdArgs []string, userOptions string) []string {
 	if !strings.Contains(userOptions, setNbdIOTimeout) {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--%s=%d", setNbdIOTimeout, defaultNbdIOTimeout))
 	}
+	if cookie != "" && util.IsVersionSatisfied(rbdNbdCookieSupportVersion, rbdNbdVersion) {
```

Comment:
check version only one and store it globally instead of checking it always?

Reply:
Aren't we doing the same now? `rbdNbdVersion` is collected once and used everytime.

Reply:
OH! Do you mean the comparison part as well?

Like move this part to global:

```
isNbdCookie = util.IsVersionSatisfied(rbdNbdCookieSupportVersion, rbdNbdVersion)

```